Formula/bitwise.rb

```diff
@@ -0,0 +1,23 @@
+class Bitwise < Formula
+  desc "Terminal based bit manipulator in ncurses"
+  homepage "https://github.com/mellowcandle/bitwise"
+  url "https://github.com/mellowcandle/bitwise/releases/download/v0.32/bitwise-v0.32.tar.gz"
+  sha256 "9dbc6646e685083cd8b876c41e828e1a4a1a6ddcb1b9765b0ef76c7fbb97f6c6"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "readline"
```

Comment:
Can this use macOS libeditline?

Reply:
Unfortunately, no. We will get many errors like `error: use of undeclared identifier 'rl_display_prompt';`. But I may miss something here.